src/System.Memory/src/System/Number/Number.NumberBuffer.cs

```diff
@@ -0,0 +1,134 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Text;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+
+namespace System
```

Comment:
Do these internal Number structs have to be in the System namespace or can they be moved to System.Buffers.Text?

Reply:
It's internal so namespace doesn't strictly matter but I'd also prefer not to introduce unnecessary diff's between these and the originals in CoreRT.